include/mbedtls/mbedtls_config.h

```diff
@@ -4080,7 +4080,7 @@
  * \def MBEDTLS_SSL_MAX_EARLY_DATA_SIZE
  *
  * The default maximum amount of 0-RTT data. See the documentation of
- * \c mbedtls_ssl_tls13_conf_max_early_data_size() for more information.
+ * \c mbedtls_ssl_conf_max_early_data_size() for more information.
```

Comment:
Seems we don't have 0-RTT(early data) in TLS 1.2, should it be enough to rename to "mbedtls_ssl_conf_tls13_early_data"? @ronald-cron-arm

Reply:
`mbedtls_ssl_conf_tls13` prefix was considered as well (see #6909) but as early data may not be TLS 1.3 specific in the future (TLS  1.4 ...) I think `mbedtls_ssl_conf` is okay.